packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1728,6 +1738,16 @@ export default class NewVm extends BaseComponent {
                 value={cpuCap}
               />
             </Item>
+            <Item label={_('cpusMax')}>
+              <DebounceInput
+                className='form-control'
+                debounceTimeout={3e3}
```

Comment:
Why?

Reply:
To let time to users to set the max `vCPUS`. In taking account that the input value is based on the max of `CPUs` and `cpusMax`.